shared/haml/user_header.haml

```diff
@@ -2,6 +2,32 @@
 - show_updated_header = request.cookies['pm'] == 'create_header_2018'
 - if current_user
   - if show_updated_header
+    .header_button.create_menu
+      %span.create_button
+        = I18n.t("nav.create_button.create")
+      &nbsp;
+      %i.create_menu_arrow_down{class: "fa fa-caret-down"}
+      %i.create_menu_arrow_up{class: "fa fa-caret-up", style: "display: none"}
+      .create_options{style: 'display: none'}
+        %a.project_link_box{href: CDO.studio_url('projects/playlab/new')}
+          %img{src: "/shared/images/fill-70x70/courses/logo_playlab.png"}
+          .project_link
+            = I18n.t("nav.create_button.playlab")
+        %a.project_link_box{href: CDO.studio_url('projects/artist/new')}
```

Comment:
Can we write some Ruby to generate these repeated options?

Reply:
Yes! Followed up in #24994